spec/controllers/idv/image_uploads_controller_spec.rb

```diff
@@ -76,6 +98,28 @@
             ]
           end
         end
+
+        it 'tracks analytics' do
+          stub_analytics
+
+          expect(@analytics).to receive(:track_event).with(
+            Analytics::IDV_DOC_AUTH_SUBMITTED_FORM,
+            {
+              success: false,
+              errors: {
+                front: [I18n.t('doc_auth.errors.not_a_file')],
+              },
+              remaining_attempts: Figaro.env.acuant_max_attempts.to_i - 1,
+            },
+          )
```

Comment:
In Ruby, if the last argument is a hash, braces are optional, so you can drop these here:

```suggestion
          expect(@analytics).to receive(:track_event).with(
            Analytics::IDV_DOC_AUTH_SUBMITTED_FORM,
            success: false,
            errors: {
              front: [I18n.t('doc_auth.errors.not_a_file')],
            },
            remaining_attempts: Figaro.env.acuant_max_attempts.to_i - 1,
          )
```

Reply:
> In Ruby, if the last argument is a hash, braces are optional, so you can drop these here:

TIL! Handy 🙂

Reply:
> In Ruby, if the last argument is a hash, braces are optional, so you can drop these here:

Updated in 02a57da.